tasks/src/zero-shot-image-classification/data.ts

```diff
@@ -3,9 +3,15 @@ import type { TaskDataCustom } from "../Types";
 const taskData: TaskDataCustom = {
 	datasets: [
 		{
-			// TODO write proper description
-			description: "",
-			id:          "",
+			description: "The STL-10 dataset is an image recognition dataset for developing unsupervised feature learning, deep learning, self-taught learning algorithms. It is similar to CIFAR-10 but with much more unlabeled examples.",
```

Comment:
```suggestion
			description: "The STL-10 dataset is an image recognition dataset for developing unsupervised feature learning, deep learning, self-taught learning algorithms. It is similar to CIFAR-10 but with much more unlabeled examples.",
```
Is it possible to make these sentences shorter, I feel like there's a bit of redundancy in them.